### ISSUE TITLE
Origin of traits species list

### ISSUE THREAD

**Issue by DeirdreLoughnan** [lizzieinvancouver/ospree#416]
@lizzieinvancouver thanks for offering to look into the selection criteria for the 96 species that we used in our trait search.

Let me know if I can help in any way.

**Comment by FaithJones** [comment]
@lizzieinvancouver  have you made progress with this issue?

**Comment by lizzieinvancouver** [comment]
@FaithJones Reminded me of this issue. We should figure this out!

@DeirdreLoughnan Hello! To help me with this can you:

- [ ] point me to this list? I did not see in traits/notes ... and I also could not find all the files in some of the cleaning code. For example, [lines 19 here](https://github.com/lizzieinvancouver/ospree/blob/master/analyses/traits/Rfiles/cleaning_bien/get.BIEN.data.R) sources a promising sounding file, but it does not seem to be on the repo ... 
- [ ] On that note, please check that all relevant files that you can upload are uploaded. This will help me track down things.

If the list were 65 species long [it would come from these criteria](https://github.com/lizzieinvancouver/ospree/issues/308) and I think it is likely output from some version of the species complex, but I cannot make much progress without some link to the list (ideally *in* the repo so I can track the file). 

Thanks!

**Comment by lizzieinvancouver** [comment]
Ooh... and in this [git issue 233](https://github.com/lizzieinvancouver/ospree/issues/233) we refer to 65 species...

**Comment by DeirdreLoughnan** [comment]
@lizzieinvancouver thanks for looking into this, the list of species is [here](https://github.com/lizzieinvancouver/ospree/blob/master/analyses/traits/notes/ospree_fullsplist.csv).

I have also updated and added the source file that was previously missing. Ultimately in looking into this issue further, it looks like we used the species list from the full OSPREE database with 234 species, but the trait databases only had trait data for about 100 of those species. At one point we discussed using the list of species from the phylogenetic analyses, but it appears that we moved forward with the longer list of species in the end. 

@lizzieinvancouver does that seem like the correct number of species from the OSPREE database?
